src/App.js

```diff
@@ -1,22 +1,16 @@
 import React from 'react';
+import { BrowserRouter, Route } from 'react-router-dom';
+import HomePage from './components/HomePage/HomePage';
+import TabsPage from './components/TabsPage/TabsPage';
 import './App.css';
 
 const App = () => (
   <div className="App">
-    <HomePage />
-    <TabsPage />
+    <BrowserRouter>
+      <Route path="/" component={HomePage} />
```

Comment:
У тебя компонент `HomePage` рендерится на всех страницах
```suggestion
      <Route path="/" exact component={HomePage} />
```